R/conda.R

```diff
@@ -506,3 +513,11 @@ conda_list_packages <- function(envname = NULL, conda = "auto", no_pip = TRUE) {
   )
   
 }
+
+conda_installed <- function() {
```

Comment:
Should this accept a `conda` argument? In case the caller of this function wants to pass a specific `conda` path.